targets/targets.json

```diff
@@ -20,6 +20,10 @@
             "network-default-interface-type": {
                 "help": "Default network interface type. Typical options: null, ETHERNET, WIFI, CELLULAR, MESH",
                 "value": null
+            },
+            "deep-sleep-latency": {
+	            "help": "Time in ms required to go to and wake up from deep sleep (max 10)",
```

Comment:
I know it's a _nitpick_ but I think you've got some tabs in this change.  The only 2 tabs in the file.